MvvmCross/Core/Core/Navigation/MvxNavigationService.cs

```diff
@@ -10,15 +10,15 @@
 using MvvmCross.Core.ViewModels;
 using MvvmCross.Core.Views;
 using MvvmCross.Platform;
+using MvvmCross.Platform.Core;
 using MvvmCross.Platform.Exceptions;
 using MvvmCross.Platform.Platform;
 
 namespace MvvmCross.Core.Navigation
 {
-    public class MvxNavigationService : IMvxNavigationService
+    public class MvxNavigationService : MvxNotifyPropertyChanged, IMvxNavigationService
```

Comment:
Why is this an `MvxNotifyPropertyChanged`? I don't see `SetProperty` called anywhere and we don't want to bake `INotifyPropertyChanged` into any implementation of `IMvxNavigationService`.